core/src/main/java/org/frankframework/lifecycle/ShowLogDirectory.java

```diff
@@ -45,10 +45,19 @@
 @IbisInitializer
 public class ShowLogDirectory {
 
-	private String defaultLogDirectory = AppConstants.getInstance().getProperty("logging.path").replace("\\\\", "\\");
-	private String defaultLogWildcard = AppConstants.getInstance().getProperty("logging.wildcard");
-	private boolean showDirectories = AppConstants.getInstance().getBoolean("logging.showdirectories", false);
-	private int maxItems = AppConstants.getInstance().getInt("logging.items.max", 500);
+	private String defaultLogDirectory;
+	private String defaultLogWildcard = AppConstants.getInstance().getProperty("log.viewer.wildcard");
+	private boolean showDirectories = AppConstants.getInstance().getBoolean("log.viewer.showdirectories", true);
+	private int maxItems = AppConstants.getInstance().getInt("log.viewer.maxitems", 500);
+
+	public ShowLogDirectory() {
+		String logdir = AppConstants.getInstance().getProperty("log.dir");
+		if(StringUtils.isEmpty(logdir)) {
+			throw new IllegalStateException("unknown log directory, property [log.dir] has not been set!");
+		}
```

Comment:
Should we also check if it exists?

Reply:
What do you mean? if it exists, it wont throw?

Reply:
If the directory to log to exists.